acapy_agent/connections/base_manager.py

```diff
@@ -7,6 +7,7 @@
 import logging
 from typing import Dict, List, Optional, Sequence, Text, Tuple, Union
 
+import pydantic
```

Comment:
Nitpick: pydantic isn't a direct dependency of ACA-Py. To catch validation errors raised by PyDID, we should instead catch `ValueError` so we don't put an implicit dependency of ACA-Py directly on pydantic.

Reply:
Good point! Will do